README.md

```diff
@@ -30,7 +32,7 @@ npm install @vinlos/ngx-gallery --save
 
 ### Versions
 
-16.x.x and above - for Angular v16+
+18.x.x and above - for Angular v18+
```

Comment:
I think we can state that old versions will continue to work with older Angular versions. So, 16+ is still valid